### PR TITLE
Update CHANGE log and RTD docs for next release

### PR DESCRIPTION
**Pull request by stsci-hack**
These changes are in preparation for the next release, including trying to get the RTD docs to build successfully.

### CODE REVIEW

File: doc/source/index.rst [hunk 1]
```diff
@@ -119,22 +119,31 @@ The task 'runastrodriz' can be used to reproduce the same Drizzle processing tha
...
 
 The code being used in the automated HST calibration pipeline to generate the products served by the HST Archive through the MAST Portal relies on multiple methods to do the best job possible in aligning and combining data regardless of whether they came from the same visit (or instrument) or not.
 
+On December 17, 2020, MAST began production of new ACS and WFC3 products in the HST data calibration pipeline: Hubble Legacy Archive (HLA)-style mosaics comprising the data from a single HST visit which are aligned to a common astrometric reference frame. These are the first of two types of **Hubble Advanced Products (HAP)** that will be produced in the HST data pipeline and made available through the `MAST Discovery Portal <https://mast.stsci.edu/portal/Mashup/Clients/Mast/Portal.html>`_.
+
+Three levels of products are available as part of this release:
+
+  * Exposure level products contain data from a single HST exposure.
+  * Filter level products are produced from all exposures in a visit with a common filter.
+  * Total level products combine all exposures from a visit and are intended as a detection image for producing catalogs.
```
Comment (mdlpstsci) [inline doc/source/index.rst:135]: Total level products...for a specific detector...

File: CHANGELOG.rst [hunk 2]
```diff
@@ -15,8 +15,26 @@ The following notes provide some details on what has been revised for each
 version in reverse chronological order (most recent version at the top
 of the list).
 
-3.2.1 (unreleased)
+3.2.2 (unreleased)
 ==================
+In addition to a couple dozen bug fixes for the new SVM processing code
+to expand the number of visits which can be aligned to GAIA, the
+following significant changes to the code or some of the API's have also
+been implemented:
```
Comment (mdlpstsci) [inline CHANGELOG.rst:23]: API's --> APIs

File: drizzlepac/align.py [hunk 3]
```diff
@@ -39,29 +39,33 @@
 # ----------------------------------------------------------------------------------------------------------
 
 
-def check_and_get_data(input_list, **pars):
+def check_and_get_data(input_list: list, **pars: object) -> list:
```
Comment (mdlpstsci) [inline drizzlepac/align.py:42]: Just curious why this function signature is setup this way now.  It is helpful, but it is also different than most functions.  As a newbie, I had to look this up!

**Review by mdlpstsci — CHANGES_REQUESTED** [review]
Just some minor changes and a question.

File: drizzlepac/align.py (hunk 3 above)
Comment (stsci-hack) [reply drizzlepac/align.py:42]: I was using PyCharm's suggestions for updating the signature of this function in an attempt to get Sphinx to successfully parse this function API.

File: CHANGELOG.rst (hunk 2 above)
Comment (stsci-hack) [reply CHANGELOG.rst:23]: Corrected in 3357098.

File: doc/source/index.rst (hunk 1 above)
Comment (stsci-hack) [reply doc/source/index.rst]: Corrected in 3357098.

**Review by mdlpstsci — APPROVED** [review]
Documentation is fine.

**Pull request merged**